app/models/models.rb

```diff
@@ -0,0 +1,66 @@
+module Qdm
```

Comment:
Quick question: do we want to capitalize all the letters in `QDM` when used as a module name? I think that's what a lot of libraries do with acronyms.

Reply:
Yeah, I'll take a look at this.